File: ios/MobilePassSDK/Model/Response/ResponseAccessPointItem.swift
//
//  ResponseAccessPointItem.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 17.02.2021.
//

import Foundation

struct ResponseAccessPointItem: Codable {
    var id:             String
    var name:           String
    var qrCodeData:     [ResponseAccessPointItemQRCodeItem]
    var geoLocation:    ResponseAccessPointItemGeoLocation?
    var deviceInfo:     ResponseAccessPointItemDeviceInfo
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/response/ResponseAccessPointItem.java
package com.armongate.mobilepasssdk.model.response;

public class ResponseAccessPointItem {
    public String id;
    public String name;
    public ResponseAccessPointItemQRCodeItem[]  qrCodeData;
    public ResponseAccessPointItemGeoLocation   geoLocation;
    public ResponseAccessPointItemDeviceInfo[]  deviceInfo;
}
